### ISSUE TITLE
Adding support for imperial units... again

### ISSUE THREAD

**Issue by devbyaccident** [smartHomeHub/SmartIR#1042]
It looks like there have been several issues raised for this already, (https://github.com/smartHomeHub/SmartIR/issues/29, https://github.com/smartHomeHub/SmartIR/issues/155, https://github.com/smartHomeHub/SmartIR/issues/160, https://github.com/smartHomeHub/SmartIR/issues/669, and https://github.com/smartHomeHub/SmartIR/issues/686) and a couple of pull requests that got closed without being merged (https://github.com/smartHomeHub/SmartIR/pull/692, https://github.com/smartHomeHub/SmartIR/pull/795) but SmartIR still doesn't have support for imperial units/Fahrenheit. 

Can one of the maintainers please add this feature? Or at least turn off stalebot while it's being worked on?

**Comment by KidA001** [comment]
Is anyone maintaining this project any more or is it dead?

**Comment by devbyaccident** [comment]
I have no idea... FWIW, I copied the contents from **climate.py** in https://github.com/smartHomeHub/SmartIR/pull/795 to my HA config, (Props to @kevin-brown, that was some solid work) and it works fine for me. Haven't out it in a branch or anything, but a forked copy would work fine.

<img width="876" alt="image" src="https://github.com/smartHomeHub/SmartIR/assets/12488263/1d21c541-4f5e-447b-96a4-6d42f94e6882">

**Comment by tcj-one** [comment]
We just had 48d7f1f 2 weeks ago, @vassilis-panos clearly has the power to merge things. So I'm not sure why #795 has been ignored.

Would love to get this implemented.

**Comment by TheAppleFreak** [comment]
Yep, I got bit by this too. It actually makes it impossible to control devices using Apple Home, as the gauge to control the temperature starts well above the max temperature in my device's codes with this incorrect conversion. 
![IMG_4863](https://github.com/smartHomeHub/SmartIR/assets/2409324/fa8f96e8-84c6-426b-9298-6cc849838b66)

**Comment by devbyaccident** [comment]
Ok, I just put in a PR for it, hope you don't mind co-author @kevin-brown, but this is long overdue.

LMK if causes issues and I'll close

**Comment by tcj-one** [comment]
We've had 3 releases since #795 and 2 since #1103, with absolutely no progress or comment on why these are being ignored. 

@vassilis-panos, can we get some response as to what's going on here?

**Comment by nelsonjchen** [comment]
If someone forks, renames, and has/keeps imperial support a priority, I
think that would likely be the most probable outcome.

**Comment by tcj-one** [comment]
It's just such a silly thing to fork over. There's no need to duplicate effort in other places to support the rest of the project in Fahrenheit. The entire change is 15 lines of code, it can and should easily coexist here.

**Comment by Pinkcommando** [comment]
Another open PR for this issue: https://github.com/smartHomeHub/SmartIR/pull/1502
